axelrod/tests/unit/test_random_.py

```diff
@@ -16,3 +17,18 @@ def test_return_values(self):
         self.assertEqual(random_choice(), C)
         random.seed(2)
         self.assertEqual(random_choice(), D)
+
+    def test_set_seed(self):
+        """Test that numpy and stdlib random seed is set by set seed helper
```

Comment:
Yeah good call.

Before I change that: what do you think about the name of the function: is `set_seed` the right way to go? Should it actually just be `seed`?

So there's `random.seed`, `numpy.random.seed` and then equivalently `axelrod.seed`?